src/TestUtils/src/DeviceTests.Runners.SourceGen/RunnerGenerator.cs

```diff
@@ -97,12 +97,15 @@ string GenerateAndroidSource()
 
 			return @"
 #if !SKIP_RUNNER_ENTRYPOINT_GENERATION && !SKIP_VISUAL_RUNNER_ENTRYPOINT_GENERATION && !SKIP_VISUAL_RUNNER_APPLICATION_GENERATION
+using System;
+using JniHandleOwnership = Android.Runtime.JniHandleOwnership;
+
 namespace " + RootNamespace + @"
 {
 	[global::Android.App.Application]
 	partial class " + appName + @" : global::Microsoft.Maui.MauiApplication
 	{
-		public " + appName + @"(global::System.IntPtr handle, global::Android.Runtime.JniHandleOwnership ownership)
+		public " + appName + @"(IntPtr handle, JniHandleOwnership ownership)
```

Comment:
The changes here, we are going to fix in:
* https://github.com/dotnet/android/pull/9680

`global::` didn't work in this new analyzer.